apps/smart-contracts/core/test/unit/DepositRecord.test.ts

```diff
@@ -237,18 +239,22 @@ describe('=> DepositRecord', () => {
       )
     })
 
-    afterEach(() => {
+    after(() => {
```

Comment:
```suggestion
    afterEach(() => {
```
Good catch!, this still needs to happen after each block however.

Reply:
Fails with afterEach